reVX/config/transmission_layer_creation.py

```diff
@@ -140,27 +149,71 @@ class MergeFrictionBarriers(BaseModel, extra='forbid'):
     by a factor. The multiplier should be large enough that all barriers have
     a higher value than any possible friction.
     """
+
+    friction_layer: str
```

Comment:
Should these be `FilePath`? That may have issues if the file doesn't exist when the script is started

Reply:
These guys are just the layer names - the path is assumed to be `<friction_layer>.tif`  if it exists on disk. This is documented in the docstring, and the reason I leave it flexible is so that you can create the friction layer on the fly without having it on disk when you launch the script.

Let me know if you see any issues with that

Reply:
👍